lib/formatters/verboseFormatter.mjs

```diff
@@ -82,6 +82,24 @@ export default function verboseFormatter(results, returnValue) {
 		printProblems('warning');
 	}
 
+	const fixedRules = getFixedRules(results);
+
+	if (fixedRules.length) {
+		let lines = '\n';
+		let total = 0;
+
+		for (const [name, count] of fixedRules) {
+			const meta = returnValue.ruleMetadata[name] || {};
```

Comment:
[suggestion] `|| {}` seems to produce an extra object because `ruleLink()` accepts `undefined` at the 2nd argument:

```suggestion
			const meta = returnValue.ruleMetadata[name];
```